ee/vellum_ee/workflows/display/nodes/vellum/inline_prompt_node.py

```diff
@@ -100,16 +100,12 @@ def serialize(
             "display_data": self.get_display_data().dict(),
             "base": self.get_base().dict(),
             "definition": self.get_definition().dict(),
-            "trigger": {
-                "id": str(self.get_target_handle_id()),
-                "merge_behavior": node.Trigger.merge_behavior.value,
-            },
+            **self.serialize_generic_fields(display_context),
```

Comment:
<img width="561" height="459" alt="Screenshot 2025-08-12 at 9 08 15 PM" src="https://github.com/user-attachments/assets/508da12b-d9b3-49b6-856a-10bc09dd7190" />

Here are all the files we want to do this for